src/modals/AddManager.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+    Box,
+    Button,
+    CircledIcon,
+    Col,
+    Input,
+    ModalWrapper,
+    Row,
+    useModal
+} from '@impact-market/ui';
+import React from 'react';
+import RichText from '../libs/Prismic/components/RichText';
+
+
+const AddManager = () => {
+    const { handleClose } = useModal();
+    
+    const handleCancel = (event: any) => {
+        event.preventDefault();
+
+        return handleClose();
+    };
+
+
+    return (
+        <ModalWrapper maxW={30.25} padding={1.5} w="100%">
+            <CircledIcon icon="flash" large />
+            <RichText
+                content="Add Manager"
+                large
+                mt={1.25}
+                semibold
+            />
+            <RichText
+                content="The new manager will receive a notification and can immediatly start adding new beneficiaries."
+                g500
+                mt={0.5}
+                small
+            />
+            {/* <form onSubmit={handleSubmit(onSubmit)}> */}
```

Comment:
If the form is currently not working (just UI finished), add a "TODO" comment so we know we have to change this later